.github/workflows/check-stage.yaml

```
@@ -0,0 +1,27 @@
+    pull_request:
+        branches:
+            - release/**
+
+jobs:
+    check-stage:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repo
+              uses: actions/checkout@v4
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)